servers/cu/src/domain/lib/loadMessageMeta.test.js

```diff
@@ -61,6 +73,18 @@ describe('loadMessageMeta', () => {
         })
         return { processId: 'process-123', timestamp: 1697574792000, nonce: 1 }
       },
+      loadProcess: async ({ processId }) => ({
+        owner: { address: 'woohoo', key: 'key-123' },
+        tags: [
+          { name: 'Module', value: 'foobar' },
+          { name: 'Data-Protocol', value: 'ao' },
+          { name: 'Type', value: 'Process' }
+        ],
+        block: { height: 123, timestamp: 1697574792000 },
+        timestamp: 1697574792000,
+        nonce: 0,
+        processId
+      }),
```

Comment:
There ought to be an additional test that triggers calling `loadProcess` stub. The stub can perform assertions on the input values and return a stubbed result.

Like `test('should load the process meta if the messageId is the processId`, ...)